src/codegen/templates/_grpc_interpreter.py.mako

```diff
@@ -157,6 +162,20 @@ class GrpcStubInterpreter(BaseInterpreter):
     def hash_task_handle(self, task_handle):
         return hash(task_handle.name)
 
+    ## get_error_string has special error handling.
+    def get_error_string(self, error_code):
+        try:
+            # Do not use self._invoke() because it may call back into self.get_error_string().
+            response = self._client.GetErrorString(
+                grpc_types.GetErrorStringRequest(error_code=error_code))
+            if not response.error_string:
+                return _UNABLE_TO_LOCATE_ERROR_RESOURCES_ERROR_MESSAGE
```

Comment:
FYI, this is another IVI dance inconsistency: if the buffer pointer is not NULL, DAQmxGetErrorString(-12345) returns "Error code could not be found...", but if the buffer pointer is NULL, it returns buffer size of 0. If grpc-device specified a non-NULL buffer pointer, it would have different behavior.

![image](https://github.com/ni/nidaqmx-python/assets/17914399/060ae822-5a3c-4973-b2bd-8286272c8e4f)